src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -12,7 +12,7 @@
 import mate.service.CarService;
 import mate.service.DriverService;
 
-@WebServlet(urlPatterns = "/drivers/add")
+@WebServlet(urlPatterns = "/cars/add")
```

Comment:
please be attentive:
```suggestion
@WebServlet(urlPatterns = "/cars/drivers/add")
```